pricing.js

```diff
@@ -28,7 +28,7 @@ function calculateVolLifePrice(product, selectedOptions) {
   const { familyMembersToCover } = selectedOptions
 
   familyMembersToCover.forEach((role) => {
-    price += calculateVolLifePricePerRole(role, selectedOptions.coverageLevel, product.costs)
+    price = calculateVolLifePricePerRole(role, selectedOptions.coverageLevel, product.costs)
```

Comment:
Not sure why you removed the `+` but you broke the function. Also you would need to preface `calculateVolLifePricePerRole` with `this.` like you did the other functions.